source/async_stack.cpp

```diff
@@ -20,13 +20,16 @@
 #include <cassert>
 #include <mutex>
 
+#if !defined(ASYNC_STACK_ROOT_USE_PTHREAD)
```

Comment:
We should prefix our macros with `UNIFEX`.
```suggestion
#if !defined(UNIFEX_ASYNC_STACK_ROOT_USE_PTHREAD)
```